tgui/packages/tgui/interfaces/BloodFilter.tsx

```diff
@@ -20,8 +19,8 @@ export const BloodFilter = (props, context) => {
             <ChemFilterPane
               title="Whitelist"
               list={whitelist}
-              reagentName={chemName}
-              onReagentInput={(value) => setChemName(value)}
+              buttonTitle="Blood White List"
```

Comment:
I can't remove buttonTitle as this variable is used by ChemFilter.tsx to display text for this button. Maybe i just do `buttonTitle="Reagent Filter List"` as the name implies reagents to filter from blood. I thought since it's about blood give it a red color. I'll give it green color instead

Reply:
Sure i'll delete the variable buttonTitle.  I thought red color is assosiated with blood but maybe  green color is nice